src/Slider.jsx

```diff
@@ -94,6 +94,21 @@ class Slider extends React.Component {
     this.onChange({ value });
   }
 
+  onKeyboard(e) {
+    const valueMutator = utils.getKeyboardValueMutator(e);
+
+    if (valueMutator) {
+      utils.pauseEvent(e);
+      const state = this.state;
+      const oldValue = state.value;
```

Comment:
If it was up to me, I would have definitely used these destructuring assignment. But I didn't want to introduce a new code styling. That's why I sticked as close as possible to the existing one. I think I based this part of the code on [this code](https://github.com/react-component/slider/blob/master/src/Range.jsx#L82-L106).